web/packages/teleterm/src/ui/Tabs/Tabs.tsx

```diff
@@ -69,7 +69,19 @@ export function Tabs(props: Props) {
     : $emptyTab;
 
   return (
-    <StyledTabs as="nav" typography="h5" bold {...styledProps}>
+    <StyledTabs
+      as="nav"
+      typography="h5"
+      bold
+      {...styledProps}
+      css={`
+        position: relative;
+        z-index: 1;
+        box-shadow: 0px 1px 10px 0px rgba(0, 0, 0, 0.12),
```

Comment:
The shadow covers a small portion of the shell content. Perhaps changing the shell component to add just a little bit of space between the tab and the shell would make the shadow not cover the shell?
 
![shell tab](https://user-images.githubusercontent.com/27113/234600133-f2241639-df35-4574-bf0f-adf66267f222.png)

Reply:
Done